src/pint/observatory/topo_obs.py

```diff
@@ -277,11 +306,11 @@ def get_dict(self):
         return {self.name: output}
 
     def get_json(self):
```

Comment:
return type?

Reply:
I was trying to apply incremental typing just to those functions I touched. But if we're going all in on type annotations then I can be more aggressive.

Reply:
ok, sure.  either way.